Roommate-Budget-Helper/budget-helper-webui
# Dev Environment Setup

## Jira Ticket: [RBH-16](https://roommatebudgethelper.atlassian.net/browse/RBH-16?atlOrigin=eyJpIjoiN2NiZDRlZWM1OTQzNGMzMDliOTI1OGIwMmI0ZmY2OTYiLCJwIjoiaiJ9)

## Description of Work
Generated the initial boilerplate for the project and got various configuration files setup
## Acceptance Criteria

- Initial boilerplate for NextJS, Tailwind, Prisma, NextAuth, and tRPC
- eslint and prettier configured
- Github actions pipeline created
- Docker-compose for local test services (Postgres database)
- Onboarded Deepsource SCA

## How to Test

- Follow setups in app/README.md and verify that the project can be run from your computer

### List of Changes:

- t3-stack boilerplate
- Readme created for setting up the project
- docker files created
